### DISCUSSION TITLE
How to access the License Server in China or restricted networks?

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#4912] [FAQs] [answered]
My Ant Media Server has been deployed in China, but the license key is always showing as invalid, preventing me from running the services smoothly. The license server is not reachable, according to the logs.

Is there a way to solve this problem?

**Comment by tremzo** [comment, marked as answer]
How to access the License Server in China or restricted networks
 - Install the proxy server
 - Configure AMS to use a Proxy Server
 - Ant Media provides a free proxy server for enterprise users. Reach out to the support channel to use it

**_Note: The proxy server is only authorized to access the license server._**

Proxy Address: proxy.antmedia.io:3199

After running the command below, the license can be verified.

`echo "proxy.address=proxy.antmedia.io:3199" >> /usr/local/antmedia/conf/red5.properties && systemctl restart antmedia`

![image](https://user-images.githubusercontent.com/121940452/226640671-95cb26d9-9e6c-44c1-86ff-469e2c841f34.png)

**Note** Please refer to this document https://antmedia.io/docs/guides/advanced-usage/activate-ams-within-restricted-geo-locations/ for more information.
